src/components/header.js

```diff
@@ -1,5 +1,7 @@
-import { graphql, useStaticQuery, Link } from 'gatsby';
+import { Link, useStaticQuery, graphql } from 'gatsby';
 import React, { useState } from 'react';
+import FCCLogo from "../assets/FCC-Nashville-blue-logo.svg"
```

Comment:
In the interests of standardization, would you mind using single quotes and a semicolon?

My linter didn't pick this one up, so that may need to be tightened up.

Reply:
Update the import statement to meet standardization.